composer.json

```diff
@@ -20,7 +20,8 @@
     "phpunit/phpunit": "~7.0",
     "mockery/mockery": "^1.1",
     "squizlabs/php_codesniffer": "^3.4",
-    "friendsofphp/php-cs-fixer": "^2.15"
+    "friendsofphp/php-cs-fixer": "^2.15",
```

Comment:
do we need this line as well?

Reply:
removed both lines, code sniffer and fixer